packages/logstash/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.5.0"
+  changes:
+    - description: Adding data and dashboards from Logstash Health Report
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/12394
```

I think you want 12464.

```suggestion
      link: https://github.com/elastic/integrations/pull/12464
```